frontend/src/firebase/firebase.js

```diff
@@ -17,5 +17,5 @@ export async function signInWithGoogle() {
 }
 
 export async function logOut() {
-  signOut(auth)
+  signOut(auth).then(() => {})
```

Comment:
Is this `.then(() => {})` necessary?

Reply:
No, but WebStorm got mad at me so I just put it in there to appease it. I'll take it out bc I don't like the way it looks